endtoend/evaluators/validator.go

```diff
@@ -116,7 +116,7 @@ func validatorsParticipating(conns ...*grpc.ClientConn) error {
 	}
 
 	partRate := participation.Participation.GlobalParticipationRate
-	expected := float32(1)
+	expected := float32(0.85)
```

Comment:
Don't forget to add a TODO issue here : )

Reply:
Got it!